incubator/jaeger/README.md

```diff
@@ -216,6 +218,10 @@ $ helm install --name myrel \
 
 Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart.
 
+If agent.hostNetwork is set, dnsPolicy needs to be set to
```

Comment:
That's a good question, but it is consistent with the collector and query specs. I can probably fix this by adding a "probably" in there. I think my case here in the docs may have been too strong--I don't know that understand all the use cases.

It may also be the case that this chart hasn't gotten much use outside of the hotrod demo app.

Reply:
I'm going to remove this paragraph entirely. The existing dnsPolicy options are not documented, nor are the existing hostPort options. I don't want to give guidance that is outside my expertise.